### ISSUE TITLE
Gradle plugin checkstyle cound not work offline

### ISSUE THREAD

**Issue by Sucran** [checkstyle/checkstyle#12352]
I have read check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```gradle

allprojects {
  apply plugin: 'checkstyle'

  checkstyle {
    configFile = rootProject.file('config/checkstyle/checkstyle.xml')
    ignoreFailures = ture
    showViolations = true
  }
}

```

excep using this plugin, I also compile the checkstyle lastest version 8.29 in the dependencies, but still not work

For Windows users, please use `type` instead of `cat` and run

```cmd
set RUN_LOCALE="-Duser.language=en -Duser.country=US"
java %RUN_LOCALE% -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
```

in place of the last 2 commands above.

---

**Describe what you expect in detail.**

---

Still not clear ???
see example - https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F

ATTENTION: FAILURE TO FOLLOW THE ABOVE TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED.

**Comment by rnveach** [comment]
@Sucran Please read over your post and https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F and follow our template to show us issue in our library.

Please note we are not maintainers of gradle plugin, so if you cannot reproduce an issue in our library, then most likely it is not our issue and is the plugin's instead.

**Comment by Sucran** [comment]
Reproduce this issue needs a env which not connecting to network. I had change the version of checkstyle, all errors points to "Cound not find host checkstyle.org" when analysing the xml file head with the dtd file's url "https://checkstyle.org/dtds/xxx.dtd". @rnveach

**Comment by rnveach** [comment]
We still need what we asked for in the template. If this can't be provided then we will have to close the issue as unable to be reproduced. I can only speculate at this point that you may not be using the correct DTD, but since nothing has been provided I cannot tell.

**Comment by Sucran** [comment]
Let me just check the dtd version whether it's compatible with the version of checkstyle. Sorry that I'm a new user of checkstyle, I had everythink ok in my laptop (which can connect to the Internet), but when I transfer all my config to my company's computer (which cannot connect to the Internet) It throws error says "could not find host checkstyle.org". I shouldn't capture the computer snapshot of the business computer since it's illegal. Maybe the error is in  the plugin of gradle, but I had not found where to report this error. @rnveach Thank you for you replay. I closed this issue firstly, and will open if I found the source of error just in checkstyle.
